### DISCUSSION TITLE
The exported react code cannot be  touch rotated with gestures on mobile browser devices

### DISCUSSION THREAD

**Discussion by zhump** [splinetool/react-spline#25] [Q&A]
The exported react code cannot be rotated with gestures on mobile browser devices。

**Comment by marcofugaro** [comment]
By default, the exported scene allows users to scroll with one finger instead of rotating the scene.

You can disable this behaviour in the `Play Settings` in the React component export panel, under the `Page Scroll` option, you have to set it to `No`.

<img width="321" alt="image" src="https://user-images.githubusercontent.com/7217420/160582825-79b9f743-c964-42ef-80e4-1f50ec15fc16.png">
